cybox/test/encoding_test.py

```diff
@@ -53,85 +56,6 @@ def test_whois(self):
         o.dnssec = UNICODE_STR
         o2 = round_trip(o)
 
-    def test_quote_xml(self):
```

Comment:
I had a moment of panicked confusion before realizing that these tests were moved to `mixbox` :)